src/Mono.Android/Android.Runtime/AndroidRuntime.cs

```diff
@@ -222,11 +231,12 @@ protected override IEnumerable<Type> GetTypesForSimpleReference (string jniSimpl
 
 		protected override IEnumerable<string> GetSimpleReferences (Type type)
 		{
-			var j = JNIEnv.GetJniName (type);
-			if (j == null)
+			var j = JNIEnv.monodroid_typemap_managed_to_java (type.FullName + ", " + type.Assembly.GetName ().Name);
```

Comment:
Wonder whether $"{type.FullName}, {type.Assembly.GetName ().Name}" wouldn't be more efficient here

Reply:
I suspect not, as `operator+` is `string.Concat()`, which can be quite efficient, as opposed to the "`string.Format()`-based" machinery of `$"..."` strings.